app/_locales/en/messages.json

```diff
@@ -2322,6 +2322,18 @@
   "errorGettingSafeChainList": {
     "message": "Error while getting safe chain list, please continue with caution."
   },
+  "errorLegalTextFirstInfo": {
+    "message": "- Technical diagnostic information."
```

Comment:
I think we might want to take the `-` bullet point out of the i18n string, and maybe even make it a real `•`